src/markdown-pages/build-apps/build-hello-world-app.mdx

```diff
@@ -275,26 +275,6 @@ Select the accounts you want to subscribe to the application, and then click **U
 
 When you return to the Apps page, you'll see the launcher for your new application.
 
-## Advance configurations
-
-### Extend webpack configuration
-
-You can use our `.extended-webpackrc.js` file to extend the webpack 4 configuration. This allows you to add your own aliases, scripts, and configurations when locally serving or deploying your app.
-
-### Proxy browser setup
```

Comment:
Is this moving to another location? It was something Miguel felt was important to bring over from the AEM site, so I actually had to go find it and add it to this doc. 😄

Reply:
Yes. I don't think this makes sense to have in "Hello, World", so it's moving. But that's in progress. That change should be in by this week 🤞 If you think it needs to stay in Hello, World until the change is in, I can add it back for now.

Reply:
Originally this change was part of that bigger restructure branch, so I overlooked that it got removed when I moved this to its own branch.

Reply:
@jpvajda I just added it back for now, so that we can get this through.